packages/uiweb/src/lib/components/chat/UserProfile/UserProfile.tsx

```diff
@@ -144,7 +144,7 @@ export const UserProfile: React.FC<UserProfileProps> = ({
           <UpdateUserProfileModal
             theme={theme}
             setModal={setShowUpdateUserProfileModal}
-            closeModalOnClickAway={closeModalOnClickAway}
+            closeProfileModalOnClickAway={closeProfileModalOnClickAway}
```

Comment:
same for the closeProfileModalOnClickAway name. Rename it to closeUserProfileModalOnClickAway